packages/manager/modules/support/src/tickets/new-ticket/feedback/feedback.service.js

```diff
@@ -1,18 +1,15 @@
+import { Environment } from '@ovh-ux/manager-config';
 import isObject from 'lodash/isObject';
 
 import { ALLOWED_LANGUAGES, BASE_URL } from './feedback.constants';
 
 export default class {
-  /* @ngInject */
-  constructor(TranslateService) {
-    this.TranslateService = TranslateService;
-  }
-
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
**question:** Couldn't be turned into a `static` method? In order to prevent disabling some ESLint rule.

Reply:
This function is the only one in a AngularJS service, used at only one place (controller in same directory).
IMHO, we could update the code to export only a `getUrl` function (es6) and avoid to use AngularJS for that.

The goal of this PR is to move the user locale management to `config` instead of `core`, could be done in another PR.